### PR TITLE
Update install text

### PR DESCRIPTION
**Pull request by keizer619**

### CODE REVIEW

**Review by praveennadarajah — APPROVED** [review]

File: resources/tools/INSTALL.txt [hunk 1]
```diff
@@ -5,8 +5,8 @@
 PREREQUISITES
 -------------
 
-Ballerina requires JRE 8 update 202.
-Check https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/tag/jdk8u202-b08 to download the JRE relevant to your system.
+Ballerina requires JRE 11.
+Check https://adoptopenjdk.net/releases.html?variant=openjdk11&jvmVariant=hotspot to download the JRE relevant to your system.
```
Comment (praneesha) [inline resources/tools/INSTALL.txt:9]:
```suggestion
Check https://adoptopenjdk.net/releases.html?variant=openjdk11&jvmVariant=hotspot to download the JRE relevant to your system.
```suggestion
[Download](https://adoptopenjdk.net/releases.html?variant=openjdk11&jvmVariant=hotspot) the JRE relevant to your system.

```
```

Comment (praneesha) [inline resources/tools/INSTALL.txt:9]:
```suggestion
Check https://adoptopenjdk.net/releases.html?variant=openjdk11&jvmVariant=hotspot to download the JRE relevant to your system.

```
Move this to connect this to the first sentence.

File: resources/tools/INSTALL.txt [hunk 2]
```diff
@@ -19,27 +19,27 @@ INSTALLING BALLERINA
 
 - Add the <BALLERINA_HOME>/bin path to your system's path variable.
 
-- If you have installed Java 8 already, you are all set! Just make sure that the JAVA_HOME environment variable is set correctly.
+- If you have installed Java 11 already, you are all set! Just make sure that the JAVA_HOME environment variable is set correctly.
 
-- If you do not have Java 8, or if you have a different version of Java, download a compatible JRE 8 version as specified in the prerequisites.
+- If you do not have Java 11, or if you have a different version of Java, download a compatible JRE 11 version as specified in the prerequisites.
```
Comment (praneesha) [inline resources/tools/INSTALL.txt:24]:
```suggestion
- If you do not have Java 11 or if you have a different version of Java, download a compatible JRE 11 version as specified in the prerequisites.

```

File: resources/tools/INSTALL.txt [hunk 3]
```diff
@@ -19,27 +19,27 @@ INSTALLING BALLERINA
...
+            ├── conf
+            ├── legal
             ├── lib
-            ├── LICENSE
             ├── man
-            ├── release
-            └── THIRD_PARTY_README
-    - Ensure that the directory which contains the JRE is named "jdk8u202-b08-jre"
+            └── release
+    - Ensure that the directory which contains the JRE is named "jdk-11.0.8+10-jre"
```
Comment (praneesha) [inline resources/tools/INSTALL.txt:36]:
```suggestion
    - Ensure that the directory, which contains the JRE is named `jdk-11.0.8+10-jre`.

```

File: resources/tools/INSTALL.txt [hunk 4]
```diff
@@ -5,8 +5,8 @@
 PREREQUISITES
 -------------
 
-Ballerina requires JRE 8 update 202.
-Check https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/tag/jdk8u202-b08 to download the JRE relevant to your system.
+Ballerina requires JRE 11.
+Check https://adoptopenjdk.net/releases.html?variant=openjdk11&jvmVariant=hotspot to download the JRE relevant to your system.
 This is the Java version we have used when testing Ballerina.
```
Comment (praneesha) [inline resources/tools/INSTALL.txt:10]:
```suggestion
This is the Java version we have used when testing Ballerina.

```
Create a single paragraph with these 3 sentences.

File: resources/tools/INSTALL.txt [hunk 5]
```diff
@@ -19,27 +19,27 @@ INSTALLING BALLERINA
 
 - Add the <BALLERINA_HOME>/bin path to your system's path variable.
```
Comment (praneesha) [inline resources/tools/INSTALL.txt:20]:
```suggestion
- Add the <BALLERINA_HOME>/bin path to your system's path variable.

```
We can add numbered bullets to these steps.

File: resources/tools/INSTALL.txt (hunk 1 above)
Comment (keizer619) [reply resources/tools/INSTALL.txt:9]: This is a text file not a MD file

**Pull request merged**